curriculum/challenges/english/01-responsive-web-design/applied-accessibility/make-screen-reader-navigation-easier-with-the-header-landmark.english.md

```diff
@@ -9,7 +9,7 @@ videoUrl: 'https://scrimba.com/c/cB76vtv'
 <section id='description'>
 The next HTML5 element that adds semantic meaning and improves accessibility is the <code>header</code> tag. It's used to wrap introductory information or navigation links for its parent tag and works well around content that's repeated at the top on multiple pages.
 <code>header</code> shares the embedded landmark feature you saw with <code>main</code>, allowing assistive technologies to quickly navigate to that content.
-<strong>Note</strong><br><code>header</code> is meant for use in the <code>body</code> tag of your HTML document. This is different than the <code>head</code> element, which contains the page's title, meta information, etc.
+<strong>Note:</strong> The <code>header</code> is meant for use in the <code>body</code> tag of your HTML document. This is different than the <code>head</code> element, which contains the page's title, meta information, etc.
```

Comment:
I have added something (in this case "The") before `<code>` because inspite of changing to the new note format, the `<p>` elements enclose `<strong>Note:</strong`> and `<code>`... separately. `<p>` is a block element, hence the lines appear in different line.
See this:
![note_code_2](https://user-images.githubusercontent.com/5875663/53896145-66fdcc80-406e-11e9-9273-b58764d24374.JPG)

After adding a word before `<code>`, the `<p>` element encloses the entire note section like this:
![note_code_1](https://user-images.githubusercontent.com/5875663/53896231-8f85c680-406e-11e9-8caa-3e6eb947aca9.JPG)

If you have any better suggestions, let me know

Reply:
Good workaround!